upp-delivery-provisioner/DEVELOPER_README.md

```diff
@@ -140,6 +143,11 @@ export KINESIS_STREAM_NAME=
 
 ## Neo4j Fleet Endpoint URL
 export NEO4J_FLEET_ENDPOINT=
+
+## Splunk API credentials
+export SPLUNK_USER=
+export SPLUNK_PASSWORD=
```

Comment:
Are the Splunk API credentials consistent across all clusters? If they are, then we should put them in the Ansible vault, so it's one less thing to have to manually set and configure when provisioning.

Reply:
We will have different values for prod and non-prod clusters, since the splunk api limitations are defined per user, and we try to avoid our requests being cut as much as we can.

Reply:
Cool, that's alright then.

On the newer provisioners we've set up separate vault files for test, prod etc - but I'm not convinced it's worth spending the time to do that for the CoCo provisioners.